src/v3/src/types/schema.ts

```diff
@@ -134,12 +135,11 @@ export function isUISchemaLayoutType(type: string): boolean {
 
 export interface FieldElement extends UISchemaElement {
   type: 'Field';
-  // TODO: only use limited field as i18n field
```

Comment:
Still needed?

Reply:
Nope this isn't really needed anymore, we have the `translations` filed in `UISchemaElement` so that solves the purpose of this TODO.